### PR TITLE
[FLOC-1845] Update gettinginvolved/contribute.html to add setup instructions.

### PR DESCRIPTION
**Pull request by abhishekmendhekar**
Update documentation for gettinginvolved/contribute.html to add setup instructions Centos7 & Ubuntu environment

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -67,6 +67,20 @@ Install Flocker's development dependencies in a ``virtualenv`` by running the fo
 .. _Docker: https://www.docker.com/
 
 
+Centos-7
+^^^^^^^^
```
Comment (davidoliverSP2) [inline CONTRIBUTING.rst]: This title should be CentOS 7 (the current style is causing a spelling related build break)

**Comment by davidoliverSP2** [comment]
Thanks @abhishekmendhekar 

One small change to the title should get this to pass the Sphinx build

**Comment by abhishekmendhekar** [comment]
Thanks Oliver. I have update this in the latest patch. I also ran the Sphinx spellcheck build to verify the changes.

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -67,6 +67,20 @@ Install Flocker's development dependencies in a ``virtualenv`` by running the fo
...
+.. prompt:: bash $
+
+   sudo yum install git python-virtualenv libffi-devel openssl-devel gcc enchant-devel
+
+Ubuntu
+^^^^^^
+
+.. prompt:: bash $
+
+   sudo apt-getting install git virtualenvwrapper python-dev libffi-dev libssl-dev enchant
```
Comment (wallrj) [inline CONTRIBUTING.rst]: `apt-getting` > `apt-get`

**Comment by sarum90** [comment]
I'm not positive that enchant is required... but I do think the docs are in better shape with this included rather than without this, so I think we should merge this as is.

Ran the ubuntu command in a VM to verify all the packages exist. Seems good.

**Pull request merged**